Collector/Collector.php

```diff
@@ -143,12 +147,52 @@ public function getClients()
      */
     public function getClientStacks($client)
     {
+        @trigger_error(sprintf('Method %s() is deprecated since version 1.8 and will be removed in 2.0. Use getClientRootStacks() instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Remove this method completely or leave it unchanged (and deprecate it). It is fine to remove it since the Collector class is `@internal`.